packages/react/ds/src/radio/radio.tsx

```diff
@@ -58,6 +59,7 @@ export const Radio = ({
           value={value}
           className={`gi-radio-base ${getRadioSize(size)}`}
           type="radio"
+          aria-label={radioId}
         />
```

Comment:
`checked` should be `defaultChecked`, else you'll have warnings all over the place when you try to change the value (about changing an uncontrolled/controlled component)

Reply:
<img width="1075" alt="image" src="https://github.com/user-attachments/assets/ea1a5237-98ab-46a2-93b8-dcecfc320c0c">

Reply:
Changing the prop name resulted in the same warning. As a solution, I tied the `readonly` attribute value to match `checked` attribute value thus ensuring that if the `Radio` has a `checked` prop it will only be a readonly